lib/cisco_node_utils/vxlan_vtep.rb

```diff
@@ -35,7 +35,7 @@ def initialize(name, instantiate=true)
     def self.vteps
       hash = {}
       is_vxlan_feature = config_get('vxlan', 'feature')
-      return hash if (:enabled != is_vxlan_feature.first.to_sym)
+      return hash if (:enabled != is_vxlan_feature.to_sym)
```

Comment:
Again.. not code I wrote, but I can refactor this.  I had suggested something simlar to Deepak because he does this in fabric path as well.  Will investigate his rational deeper and see if it can be refactored.